src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/SecuredPersistentObject.java

```diff
@@ -29,12 +27,10 @@
 	private static final long serialVersionUID = 1L;
 
 	@OneToMany
```

Comment:
Please change this to `@ManyToMany` as we figured out that `@OneToMany` is wrong here.